package.json

```diff
@@ -14,6 +14,7 @@
     "axios": "0.19.0",
     "bootstrap": "4.4.1",
     "bootstrap-vue": "2.2.2",
+    "bootswatch": "^4.4.1",
```

Comment:
I'd prefer to use fixed version instead. Can you remove the `^` plz ?

Reply:
It's fixed :)